_data/url-redirects.json

```diff
@@ -122,5 +122,6 @@
 	"testing-your-app-inside-the-opera-tv-store": "tv/testing-your-app-inside-opera-tv-store",
 	"tweaking-spatial-navigation-for-tv-browsing": "tv/tweaking-spatial-navigation-for-tv-browsing",
 
-	"web-notifications-in-opera-developer-24": "web-notifications-in-opera-developer-25"
+	"web-notifications-in-opera-developer-24": "web-notifications-in-opera-developer-25",
+	"mini": "blog/opera-mini-server-upgrade"
```

Comment:
This would generate the following rule, which is not what we want:

```
RewriteRule ^articles/view/mini/?$ blog/opera-mini-server-upgrade/ [R=301,L]
```